tests/js/browser-binder-tests.js

```diff
@@ -0,0 +1,639 @@
+// Test "binder" components using `gpii-test-browser`.
+"use strict";
+var fluid = require("infusion");
+var gpii  = fluid.registerNamespace("gpii");
+
+require("gpii-test-browser");
+gpii.tests.browser.loadTestingSupport();
+
+var path    = require("path");
+var url     = require("url");
+var fileUrl = url.resolve("file://", path.resolve(__dirname, "../static/tests-binder.html"));
+
+fluid.registerNamespace("gpii.binder.tests");
+
+// TODO:  If we find ourselves making model changes often, move this to `gpii-test-browser` and polish it up for wider use.
+gpii.binder.tests.applyChange = function (componentPath, modelPath, newValue) {
+    var component = fluid.getGlobalValue(componentPath);
+    component.applier.change(modelPath, newValue);
+};
+
+
+fluid.defaults("gpii.binder.tests.caseHolder", {
```

Comment:
@amb26 and @the-t-in-rtf thanks for including me in this thread. I'd be interested to see where it goes. Intern looks interesting, but seems like a solution that would replace qUnit. Although we might be able to keep many of the jqUnitisms we've developed over the years, and there are things we'd probably like to improve over qUnit. However, it would likely be a lot of work to do that refactoring. Were you thinking about trying to just take out the parts needed for simulating events, or making use of it as a whole?